model/telemetry/entities.yaml

```diff
@@ -8,10 +8,13 @@ groups:
     attributes:
       - ref: telemetry.sdk.name
         requirement_level: required
+        role: identifying
```

Comment:
package name doesn't seen unique across languages, should language be identifying as well?